jsonschema-core.md

```diff
@@ -2324,9 +2322,9 @@ allows their users to craft their own messages.
 
 The passing instance will produce the following annotations:
 
-- The keyword `title` evaluated at `` by following the path `` will produce
```

Comment:
This rendered weird in VSCode, but I wasn't sure if it was a problem for the "publish" render engine.  VSCode rendered it with " by folloing the path" as monospace text instead of recognizing that `` is an empty string.  (Not really sure how do formatting on an empty string.)

My solution was to just reword it.

Reply:
It looks like the remark build has the same problem.